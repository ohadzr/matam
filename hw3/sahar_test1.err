Error: trainer does not exist.
Error: trainer does not exist.
Error: invalid argument.
Error: invalid argument.
Error: trainer name already exist.
Error: location does not exist.
Error: invalid argument.
Error: invalid argument.
Error: invalid argument.
Error: location is not reachable.
Error: location does not exist.
Error: location is not reachable.
Error: location is not reachable.
Error: trainer already in location.
Error: budget is not sufficient.
Error: no available item found.
Error: item out of stock.
Error: no available item found.
Error: item out of stock.



