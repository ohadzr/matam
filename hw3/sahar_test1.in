
report locations
trainer go Ohad paris
trainer go Ohad palet

trainer add Ohad 100 paris
trainer add Shani 100 sydny
trainer add Gal 200 hong-kong
trainer add Sahar 400 mumbai
trainer add Itay 0 mumbai
trainer add Tamar -1 paris
trainer add Tomer 100 london
trainer add Tomer -1 palet
trainer add Tomer 1 palet
trainer add Dana 1 palet

store add candy 10 -5
store add candy -10 5
store add item 10 5
store add candy 10 5
store add candy 20 5
store add candy 30 1
store add candy 40 5
store add potion 20 2
store add potion 30 1
store add potion 70 1
store add potion 100 2
store report

trainer go Ohad sydny
trainer go Ohad japan
trainer go Ohad london
trainer go Shani london
trainer go Shani madrid
trainer go Sahar alexandria
trainer go Sahar saint-petersburg
trainer go Ohad london
trainer go Ohad paris
trainer go Ohad london
trainer go Itay shanghai 
trainer go Itay berlin 

triner purchase Itay candy 30 
triner purchase Itay potion 10 
triner purchase Gal candy 100
triner purchase Gal candy 30
triner purchase Gal potion 100
triner purchase Ohad potion 10
triner purchase Ohad potion 20
triner purchase Ohad potion 20
triner purchase Ohad potion 20
store add potion 20 1
triner purchase Ohad potion 20

trainer go Tomer alexandria
report locations
battle fight Ohad Shani charmander caterpie
battle fight Itay Tomer Dragonite Mew




