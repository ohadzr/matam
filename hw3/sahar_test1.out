** location report **
Ohad captured a pidgey in paris.
Shani captured a caterpie in sydny.
hong-kong is empty: Gal could not catch any pokemons.
Sahar captured a Mewtwo in mumbai.
mumbai is empty: Itay could not catch any pokemons.
Tomer captured a pidgey in london.
** store report **
Ohad captured a pikachu in london.
Sahar captured a Cyndaquil in alexandria.
Sahar captured a Dragonair in saint-petersburg.
Ohad captured a rattata in paris.
Ohad captured a charmander in london.
Itay captured a Dragonite in shanghai.
Itay captured a ivysaur in berlin.
Tomer captured a Mew in alexandria.
** location report **
** 2 fights **