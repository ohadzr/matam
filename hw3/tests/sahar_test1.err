Error: trainer does not exist.
Error: trainer does not exist.
Error: invalid argument.
Error: invalid argument.
Error: trainer name already used.
Error: location does not exist.
Error: invalid argument.
Error: invalid argument.
Error: invalid argument.
Error: location is not reachable.
Error: location does not exist.
Error: location is not reachable.
Error: location is not reachable.
Error: trainer already in location.
Error: budget is insufficient.
Error: requested item is out of stock.
Error: requested item is out of stock.
Error: requested item is out of stock.
Error: requested item is out of stock.



