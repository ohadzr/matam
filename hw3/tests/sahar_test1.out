
************  Locations Report  ***********
alexandria          :  Cyndaquil
berlin              :  ivysaur
hong-kong           :  empty
london              :  pidgey
madrid              :  bulbasaur
mumbai              :  Mewtwo
paris               :  pidgey
rome                :  pikachu
saint-petersburg    :  Dragonair
shanghai            :  Dragonite
sydney              :  caterpie
tokyo               :  Typhlosion
Ohad captured a pidgey in paris.
Shani captured a caterpie in sydney.
hong-kong is empty: Gal could not catch any pokemons.
Sahar captured a Mewtwo in mumbai.
mumbai is empty: Itay could not catch any pokemons.
Tomer captured a pidgey in london.

***   Stock Report   ***
candy    10             X 5
candy    20             X 5
candy    30             X 1
candy    40             X 1
potion   20             X 2
potion   30             X 1
potion   70             X 1
potion   100            X 2
Ohad captured a pikachu in london.
Sahar captured a Cyndaquil in alexandria.
Sahar captured a Dragonair in saint-petersburg.
Ohad captured a rattata in paris.
Ohad captured a charmander in london.
Itay captured a Dragonite in shanghai.
Itay captured a ivysaur in berlin.
Tomer captured a Mew in alexandria.

************  Locations Report  ***********
alexandria          :  raichu
berlin              :  squirtle
hong-kong           :  empty
london              :  pidgey
madrid              :  bulbasaur
mumbai              :  empty
paris               :  pikachu
rome                :  pikachu
saint-petersburg    :  Dratini
shanghai            :  butterfree
sydney              :  pikachu
tokyo               :  Typhlosion

************  Pokemon Battle  *************
Ohad             		vs.		            Shani
charmander       		   		         caterpie
15               		CP 		               10
100.00 --> 85.00 		HP 		100.00 -->  77.50
1      --> 2     		LVL		     1 -->      2
1.00   --> 3.25  		XP 		  1.00 -->   2.50

************  Pokemon Battle  *************
Itay             		vs.		            Tomer
Dragonite        		   		              Mew
100              		CP 		              500
100.00 --> 0.00  		HP 		100.00 -->   0.00
1      --> 2     		LVL		     1 -->      2
1.00   --> 16.00 		XP 		  1.00 -->  76.00
Itay's Dragonite has died in the battle.
Tomer's Mew has died in the battle.
Gal captured a pikachu in rome.

************  Pokemon Battle  *************
Gal              		vs.		            Shani
pikachu          		   		         caterpie
60               		CP 		               10
100.00 --> 62.50 		HP 		100.00 -->  10.00
1      --> 2     		LVL		     2 -->      3
1.00   --> 10.00 		XP 		  2.50 -->   6.25

*************  Trainer Report  ************
Gal
Currently in rome
Budget  60    Pokecoins
XP      10.00

* Trainer's Items:
potion   100            X 1
* Trainer's Pokemons:
1    pikachu             HP     62.50
                         CP        60
                         LEVEL      2

*************  Trainer Report  ************
Tomer
Currently in alexandria
Budget  130   Pokecoins
XP      76.00

* Trainer's Items:
* Trainer's Pokemons:
1    pidgey              HP    100.00
                         CP        20
                         LEVEL      1
